benchmarks/nemo/nemolite2d/manual_versions/psykal_cpp/Makefile

```diff
@@ -0,0 +1,88 @@
+# Makefile for manual OpenMP version of PSyKAl'd NEMOLite2D.
+#
+# This Makefile picks up the compiler to use any flags from
+# environment variables. e.g. to use gfortran:
+# 
+# export F90=gfortran
```

Comment:
For clarity please could you add CXX to this list of environment variables.

Reply:
Done, also added CFLAGS